kong/clustering/rpc/manager.lua

```diff
@@ -202,6 +195,19 @@ function _M:_handle_meta_call(c, node_id)
     return nil, err
   end
 
+  local capabilities_list = info.rpc_capabilities
+
+  self.client_capabilities[node_id] = {
+    set = pl_tablex_makeset(capabilities_list),
+    list = capabilities_list,
+  }
+
+  -- store DP's ip addr
+  self.client_info[node_id] = {
+    ip = ngx_var.remote_addr,
+    version = info.kong_version,
+  }
```

Comment:
Should `_handle_meta_call` also `assert(self.concentrator)`? In case `self.client_info` is not set.

Reply:
added.